src/js/dropdown-menu/dropdown.underscore

```diff
@@ -1,5 +1,11 @@
-<a href="<%- main.url %>" class="user-title">
-    <span class="sr"><%- main.screenreader_label %> </span><%- main.text %>
+<a href="<%- main.url %>" class="menu-title">
+    <% if (main.screenreader_label) { %>
```

Comment:
It's to provide context to the text. So in our marketing site implementation it is 'Dashboard for: ' and the text is 'username'. So maybe 'screenreader_context' could be a more meaningful name?